### PR TITLE
Touch ups

### PR DESCRIPTION
**Pull request by EdThePro101**
I have done multiple touch-ups to organize the README file so that it can be read easily.

Also, I have updated the year in the LICENSE file as it was a year behind (2018).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,205 +1,241 @@
...
-- Identify keywords from millions of text questions
-- Dataset: [StackOverflow Questions Sample by Facebook](https://www.kaggle.com/c/facebook-recruiting-iii-keyword-extraction/data)
+- [Recommendation Systems](#recommendation-systems)
 
-**Topics Identification**
-- Multi-label classification of printed media articles to topics
-- Dataset: [Greek Media Monitoring Multilabel Classification](https://www.kaggle.com/c/wise-2014/data)
+- [Vision](#vision) - With topics about image and video processing
+
+- [Music and Audio](#music) - These topics are bout combining ideas from language and audio to understand music
```
Comment (NirantK) [inline README.md]:
```suggestion
- [Music and Audio](#music) - These topics are about combining ideas from language and audio to understand music
```
Fix spelling typo

File: README.md [hunk 2]
```diff
@@ -1,205 +1,241 @@
...
+
+- [Conclusion](#conclusion)
+
+---
+
+## Text <a name="text"></a>
```
Comment (NirantK) [inline README.md]: The internal hyperlinks work natively with Github Markdown. Why add the <a></a> tags?

Comment (EdThePro101) [reply README.md]: My editor automatically noticed them and changed them. Not really sure why though...

Comment (NirantK) [reply README.md]: Can we please remove those and send a new PR?

Comment (EdThePro101) [reply README.md]: Yes sure. Must I remove just the tags?

**Pull request closed without merging**